components/Tabs/Tabs.js

```diff
@@ -2,50 +2,56 @@
 class TabLink {
   constructor(element) {
     // Assign this.element to the passed in DOM element
-    // this.element;
+    this.element = element; 
     
     // Get the custom data attribute on the Link
-    // this.data;
+    this.data = element.dataset.tab; 
     
     // Using the custom data attribute get the associated Item element
-    // this.itemElement;
+    this.itemElement = document.querySelector(`.tabs-item[data-tab="${this.data}"]`); 
     
     // Using the Item element, create a new instance of the TabItem class
-    // this.tabItem;
+    this.tabItem = new TabItem(this.itemElement); 
     
     // Add a click event listener on this instance, calling the select method on click
-
+        this.element.addEventListener('click', event => {
+        this.select()
+      })
   };
 
   select() {
     // Get all of the elements with the tabs-link class
-    // const links;
+    const links = document.querySelectorAll(".tab-links")
 
     // Using a loop or the forEach method remove the 'tabs-link-selected' class from all of the links
-    // Array.from(links).forEach();
+    links.forEach(element  => {
+      links.classList.remove("tabs-links-selected"); 
```

Comment:
Make sure to remove `"tabs-links-selected"` from each `element` in your links iteration instead of the `links` NodeList